Makefile

```diff
@@ -18,8 +19,9 @@ endif
 include $(SHIPYARD_DIR)/Makefile.inc
 
 TARGETS := $(shell ls -p scripts | grep -v -e /)
-export LDFLAGS = -X main.VERSION=$(VERSION)
-
+GIT_COMMIT_HASH := $(shell git show -s --format='format:%H')
+GIT_COMMIT_DATE := $(shell git show -s --format='format:%aI')
```

Comment:
Format "%aI" is the author date which is not very useful. The interesting date is the commit date, when the commit became available to the world, so "%cI".

Reply:
Thanks.